### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dmitruz**
- [DEMO LINK](https://dmitruz.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://dmitruz.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$color-light: #fff;
+$color-special: #F3F3F3;
+$color-main: #616070;
+$color-main-accent: #060B35;
+$color-strong-accent: #00ACDC;
```
Comment (volodymyr-soltys97) [inline src/styles/variables.scss:5]:
```suggestion
$color-strong-accent: #00ACDC;

```

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import 'variables';
+@import 'blocks/settings';
+@import 'blocks/card';
+@import 'blocks/stars';
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:4]:
```suggestion
@import 'blocks/stars';

```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
Add hover effects for button when hovering only button, not all card
image

**Review by nazarmatsevych — APPROVED** [review]
🔥

**Pull request closed without merging**